File: go/iq/md~set~companion_platform_display+companion_platform_id+companion_server_auth_key_pub+l-916c365c.go
// BuildMdCompanionHello builds <iq type="set" xmlns="md">. Replies with one of: NotifyCompanion, Error.
func BuildMdCompanionHello(jid types.JID, shouldShowPushNotification string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "md", "to": "s.whatsapp.net", "type": "set"},
		Content: []waBinary.Node{
			{Tag: "link_code_companion_reg", Attrs: waBinary.Attrs{"jid": jid, "stage": "companion_hello", "should_show_push_notification": shouldShowPushNotification}},
		},
	}
}

File: ts/iq/md~set~companion_platform_display+companion_platform_id+companion_server_auth_key_pub+l-916c365c.ts
/** <iq type="set" xmlns="md">. Replies with one of: NotifyCompanion, Error. */
export function buildMdCompanionHello(jid: string, shouldShowPushNotification?: string): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'md', to: 's.whatsapp.net', type: 'set' }, content: [
      { tag: 'link_code_companion_reg', attrs: { jid: jid, stage: 'companion_hello', should_show_push_notification: shouldShowPushNotification } },
    ] }
}